mso/resource_mso_schema_site_bd.go

```diff
@@ -71,22 +77,25 @@ func resourceMSOSchemaSiteBdImport(d *schema.ResourceData, m interface{}) ([]*sc
 	if err != nil {
 		return nil, fmt.Errorf("No Sites found")
 	}
-	stateSite := get_attribute[2]
+	site_template := regexp.MustCompile("-").Split(get_attribute[2], 2)
```

Comment:
Why make this more complex? Shall we consider {schema_id}/site/{site_id}/template/{template_name}/bd/{bd_name} as input of import statement? Think this will make the import statement and code easier.